src/index.html

```diff
@@ -4,9 +4,443 @@
+    <a
+       href="#Home"
+       class="header__home">
+      <img
+        src="images/logo.png"
+        alt="Home"
+      >
+    </a>
+    <nav class="header__nav">
+      <ul class="header__list">
+        <li class="header__item">
+          <a
+            href="#apple"
+            class="header__link header__link--active is-active"
+          >
+            Apple
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#samsung"
+            class="header__link"
+          >
+            Samsung
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+             href="#smartphones"
+             class="header__link"
+          >
+             Smartphones
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#laptops&computers"
+            class="header__link"
+            data-qa="nav-hover"
+          >
+            Laptops & Computers
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#gadgets"
+            class="header__link"
+          >
+            Gadgets
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#tablets"
+            class="header__link"
+          >
+            Tablets
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#photo"
+            class="header__link"
+          >
+            Photo
+          </a>
+        </li>
+
+        <li class="header__item">
+          <a
+            href="#video"
+            class="header__link"
+          >
+            Video
+          </a>
+        </li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <div class="container">
+      <div class="container__card card" data-qa="card">
+        <div class="card__image">
+          <img
+            class="card__monoblock"
+            src="images/imac.jpeg"
+            alt="monitor"
+          >
+        </div>
+
+        <div class="card__title">
+          <p class="card__title-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+          <p class="card__title-code">Product code: 195434</p>
+        </div>
+
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-text">Reviews: 5</p>
+
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-value">$2,199</p>
+        </div>
+
+        <div class="card__buy card__buy--hover">
+          <a
+            href="#"
+            class="card__buy-link card__buy-link--hover"
+            data-qa="card-hover"
+
+          >
```

Comment:
```suggestion
          <a
            href="#"
            class="card__buy-link"
            data-qa="card-hover"
          >
```
Do you really need to add hover modifier? You can just use a hover pseudoclass for card__buy-link